planning_trajectory_verifier:
  min_colliding_time: 5

online_trajectory_verifier:
  min_colliding_time: 5

global_goal_trajectory_scoring:
  height_cost:
    weight: 1
    max_cost: 1
    max_height: 2.25
    des_height: 1.5
    min_height: 0.75

  obstacle_cost:
    desired_freespace: 0.5
    weight: 10

current_traj_cost_factor: 0

end_point_trajectory_source:
  max_traj_duration: 10
  future_dater:
    delay_duration: 0
  ni_params:
    c_p: 0.2
    c_d: 3

local_goal_trajectory_source:
  max_traj_duration: 10
  future_dater:
    delay_duration: 0
  ni_params:
    c_p: 0.2
    c_d: 3

local_search_trajectory_source:
  max_traj_duration: 5
  future_dater:
    delay_duration: 0

replan_logic:
  max_replan_period: -1
  min_tte: 5

feedback_publish_freq: 5

